homely: init at 0.15.3

lokke-me opened the pull request:
Added package meta

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

LnL7 committed:
linux-headers: Fix Darwin cross build

Carefully fake cc-version and cc-fullversion to avoid needing a compiler
for the kernel itself to build the headers.

For some reason, doing `make install_headers` twice, first without
INSTALL_HDR_PATH=$out then with, is neccessary to get this to work.

Ericson2314 committed:
release-cross: Cross compile from all 3 supported platforms

roberth committed:
Add Nixpkgs functions for adding overlays ad-hoc

This is something that I have found useful in tests. In practice
I recommend that people call Nixpkgs once for performance and
simplicity. The inline documentation mentions this too.

peterhoeg committed:
firejail: override files should be in /etc/firejail

teto committed:
linux: inside nix-shell, allow make menuconfig

Use stdenv.lib.inNixShell to check for nix-shell and potentially bring pkgconfig/ncurses.

peterhoeg committed:
snapcast: 0.14.0 -> 0.15.0

roberth committed:
nixpkgs docs: Point to pkgs.extend, pkgs.appendOverlays and improve override section

r-ryantm committed:
clamav: 0.100.1 -> 0.100.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/clamav/versions

r-ryantm committed:
aws-sdk-cpp: 1.5.17 -> 1.6.20

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/aws-sdk-cpp/versions

vbgl committed:
coqPackages.category-theory: fix build

baracoder committed:
nvidia_x11: 390.87 -> 410.66 for x86_64

thoughtpolice committed:
cloudflared: init at 2018.10.3

'cloudflared' is a multi-purpose client-side tool for CloudFlare Argo
Tunnel, CloudFlare Access, as well as including a simple DNS-over-HTTP
(DoH) proxy tool as well.

However, 'cloudflared' is NOT available under an open source license.
Furthermore, the exact terms of redistribution (namely, if we are able
to redistribute binaries at all) are not entirely clear to me. As a
result, I have filed the following bug report concerning the terms of
redistribution for the source code and binaries:

  https://github.com/cloudflare/cloudflared/issues/53

'cloudflared' does have source code available, however, and it
encourages users to use 'go install' in order to set it up, in fact (or
download their prebuilt, compiled binaries). So using the source seems
to be encouraged. Even then, I'm still not sure if Hydra can serve these
binaries.

In lieu of a more pointed answer regarding source/binary licensing, and
to avoid keeping this expression in my private tree, I've marked it as
'unfree' (to avoid Hydra serving it in any way) as well as compiled from
source (to avoid any 'redistribution allowed while unmodified' terms
that may crop up).

The dependencies for this build were generated using 'dep2nix'.

Signed-off-by: Austin Seipp <aseipp@pobox.com>

roberth committed:
doc: Improve overrides and overlays documentation

dtzWill committed:
libinput-gestures: 2.33 -> 2.38

dtzWill committed:
libinput-gestures: update patches

Don't bother skipping the version check,
keep diff against upstream simple.

dtzWill committed:
libinput-gestures: entry point is a python script, add dep

Otherwise we have a raw shebang that doesn't work :(

vdemeester committed:
skaffold: init at 0.16.0

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

roberth committed:
doc: Add link from overrides to overlays

alyssais committed:
gem-config: capybara-webkit: qt48 -> qt59

> WARNING: The next major version of capybara-webkit will require at
> least version 5.0 of Qt. You're using version 4.8.7.

I went to 5.9 instead of 5.11 because 5.11 doesn't currently build on
Darwin, whereas 5.9 can build on both Darwin and Linux, and is still
well within the >=5.0 requirement.

marsam committed:
redis: 4.0.11 -> 5.0.0

markuskowa committed:
samba: remove defunct infiniband support

Even if infiniband support is enabled the option
is ignored by the configure script and no depencies
to libibverbs.so or librdmacm.so is contained in
the output.

vdemeester committed:
kube-prompt: init at 1.0.4

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

danieldk committed:
openal: 1.19.0 -> 1.19.1

Changes: https://github.com/kcat/openal-soft/blob/master/ChangeLog

Also fetch from kcat's openal-soft official GitHub repository, since
upstream's website http://kcat.strangesoft.net/openal.html is down.

mpickering committed:
cedille: Apply patch which unbreaks the options file

See https://github.com/cedille/cedille/issues/29

mpickering committed:
cedille: Make a seperate output for the base library

For now, for normal usage you need to add `${cedille.lib}` to
the `~/.cedille/options` file so that cedille will find the base
libraries. There is no option to pass options by the command line.

loskutov committed:
opensc: 0.18.0 -> 0.19.0

Zimmi48 committed:
ocamlformat: init at 0.8

nyanloutre committed:
jackett: 0.10.304 -> 0.10.365

Commit:
nvidia_x11_beta: Set to nvidia_x11

There is currently no beta active

dhess committed:
dovecot: dovenull user should have its own group.

Quoting from https://wiki.dovecot.org/UserIds#dovenulluser:

"It should belong to its own private dovenull group where no one else
belongs to..."

lokke-me committed:
homely: init at 0.15.3

Added package meta

lokke-me committed:
homely: init at 0.15.3

Added package meta

requested changes

homely: init at 0.15.3

Added package meta

r-ryantm committed:
libowfat: 0.31 -> 0.32 (#48414)

* libowfat: 0.31 -> 0.32

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/libowfat/versions

* libowfat: keep headers under include directory

0.32 wanted to move them under include/libowfat/ 
but it breaks gatling and opentracker

primeos committed:
wlroots: 2018-03-16 -> 0.1

Keep "meta.broken = true" but update the package to keep track of
upstream changes and prepare the "reintegration" into nixpkgs.

bhipple commented on pkgs/applications/misc/homely/default.nix:
```diff
@@ -0,0 +1,33 @@
+  propagatedBuildInputs = with pythonPackages; [ requests
+                        simplejson
+                        click
+                        pythondaemon
```
As a heads up, I may rename this to its pypi name in https://github.com/NixOS/nixpkgs/pull/47897, pending feedback from @FRidh.

It's cosmetic, but I'd also put requests on the newline to fixup your whitespace a little.

bhipple commented on pkgs/top-level/all-packages.nix:
```diff
@@ -22724,4 +22724,6 @@ with pkgs;
 
   tsung = callPackage ../applications/networking/tsung {};
+
+  homely = callPackage ../applications/misc/homely {};
```
We generally try to keep this file somewhat sorted within sections.

Mic92 committed:
Merge pull request #48834 from dhess/dovenull-group-fix

dovecot: dovenull user should have its own group.

veprbl committed:
zstd: 1.3.6 -> 1.3.7

jwiegley committed:
wabt: Allow building on Darwin as well

dywedir committed:
Merge pull request #48846 from veprbl/pr/zstd_137

zstd: 1.3.6 -> 1.3.7

r-ryantm committed:
zimg: 2.7.5 -> 2.8

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/zimg/versions

dywedir committed:
skim: 0.5.1 -> 0.5.2

r-ryantm committed:
wavebox: 4.3.0 -> 4.4.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/wavebox/versions

nlewo committed:
Merge pull request #48614 from vdemeester/init-kube-prompt

kube-prompt: init at 1.0.4

7c6f434c committed:
lrs: init at 7.0

srhb committed:
Merge pull request #48783 from marsam/feature/update-redis

redis: 4.0.11 -> 5.0.0

vdemeester committed:
buildah: 1.1 -> 1.4

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

Mic92 committed:
Merge pull request #48859 from dywedir/skim

skim: 0.5.1 -> 0.5.2

joachifm committed:
Merge pull request #48855 from r-ryantm/auto-update/zimg

zimg: 2.7.5 -> 2.8

7c6f434c committed:
Merge pull request #48808 from danieldk/openalSoft-1.19.1

openal: 1.19.0 -> 1.19.1

peterhoeg committed:
Merge pull request #47718 from peterhoeg/f/fj

firejail: override files should be in /etc/firejail

nyanloutre committed:
jackett: update mono version to 5.14

See details here : https://github.com/NixOS/nixpkgs/pull/48821#issuecomment-431837095

7c6f434c committed:
visidata: init at 1.3.1

etu committed:
sequeler: 0.6.2 -> 0.6.3 (#48858)

Changelog: https://github.com/Alecaddd/sequeler/releases/tag/v0.6.3

Mic92 committed:
Merge pull request #48860 from r-ryantm/auto-update/wavebox

wavebox: 4.3.0 -> 4.4.0

Mic92 committed:
Merge pull request #48821 from nyanloutre/jackett-10-365

jackett: 0.10.304 -> 0.10.365

infinisil committed:
Merge pull request #48661 from baracoder/nvidia-update

nvidia_x11: 390.87 -> 410.66

kalbasit committed:
gocode: 2018-07-27 -> 2018-10-22 (#48832)

github.com/nsf/gocode has also been deprecated in favor of github.com/mdempsky/gocode

alyssais committed:
weechat: add perl.withPackages (#48815)

Allows for adding Perl libraries in the same way as for Python. Doesn't
really need to be a function, since there's only one perlPackages in
nixpkgs, but I went for consistency with the python plugin.

Synthetica9 committed:
wlroots: remove broken tag

See: https://github.com/NixOS/nixpkgs/issues/38344#issuecomment-432032834

rbvermaa committed:
datadog-agent: add option to enable trace agent

eadwu committed:
vscode-extensions.ms-python.python: 2018.8.0 -> 2018.9.1 (#48849)

magnetophon committed:
dragonfly-reverb: 0.9.1 -> 0.9.4 (#48691)

xeji committed:
Merge pull request #48869 from Synthetica9/patch-2

wlroots: remove broken tag

Mic92 committed:
Merge pull request #48856 from peterhoeg/u/sc

snapcast: 0.14.0 -> 0.15.0

mguentner committed:
gem-config: set libmagic for magic

r-ryantm committed:
snd: 18.7 -> 18.8

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/snd/versions

joachifm committed:
Merge pull request #48794 from markuskowa/fix-samba-ib

samba: remove defunct infiniband support

joachifm committed:
Merge pull request #48541 from r-ryantm/auto-update/clamav

clamav: 0.100.1 -> 0.100.2

iblech committed:
nixpkgs docs: Fix tiny typo (#48477)

joachifm committed:
Merge pull request #47909 from teto/menuconfig

linux: inside nix-shell, allow make menuconfig

r-ryantm committed:
sysstat: 12.0.1 -> 12.1.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/sysstat/versions

r-ryantm committed:
smplayer: 18.9.0 -> 18.10.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/smplayer/versions

Mic92 committed:
Merge pull request #48888 from alyssais/capybara-webkit

gem-config: capybara-webkit: qt48 -> qt59

r-ryantm committed:
sleuthkit: 4.6.2 -> 4.6.3

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/sleuthkit/versions

Mic92 committed:
Merge pull request #48886 from r-ryantm/auto-update/sysstat

sysstat: 12.0.1 -> 12.1.1

r-ryantm committed:
tortoisehg: 4.7.1 -> 4.7.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/tortoisehg/versions

r-ryantm committed:
star: 2.6.1a -> 2.6.1c

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/star/versions

rbvermaa committed:
opentracing-cpp: init 1.5.0

(cherry picked from commit f49df04c02d083395957fc85e12d231bb536e451)

bachp committed:
gitlab-runner: 11.3.1 -> 11.4.0 (#48867)

rbvermaa committed:
opentracing-cpp: add meta

(cherry picked from commit 0b695b6682a5e8d391a5dcf81329ddec527c0eb5)

rbvermaa committed:
nginx-opentracing: init at 0.7.0

(cherry picked from commit 9d6184f1bcd2a57db31c48fd0e1847147d44715a)

rbvermaa committed:
opentracing-cpp: fix version in derivation

(cherry picked from commit df29219f5118dd1ae1d02108aeea4484e20d67e8)

r-ryantm committed:
sslscan: 1.11.11 -> 1.11.12

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/sslscan/versions

r-ryantm committed:
sundials: 3.2.0 -> 3.2.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/sundials/versions

mmcalistair committed:
Merge branch 'homely' of github.com:lokke-me/nixpkgs into homely

r-ryantm committed:
python27Packages.rpm: 4.14.2 -> 4.14.2.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/rpm/versions

markuskowa committed:
libuv: add licenses

markuskowa committed:
libsigc++: add license

dtzWill committed:
brotli: 1.0.6 -> 1.0.7 (#48874)

magnetophon committed:
caps: 0.9.24 -> 0.9.26 (#48889)

markuskowa committed:
libwmf: add license

markuskowa committed:
libwpd: add meta data

alyssais committed:
perlPackages.XMLFeedPP: init at 0.95 (#48816)

markuskowa committed:
libwpg: add licenses

markuskowa committed:
log4shib: add license + homepage

markuskowa committed:
loudmouth: add license

r-ryantm committed:
shairport-sync: 3.2.1 -> 3.2.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/shairport-sync/versions

markuskowa committed:
martyr: add license

markuskowa committed:
mdds: add license

markuskowa committed:
mono-addins: add license

r-ryantm committed:
rocksndiamonds: 4.1.0.0 -> 4.1.1.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/rocksndiamonds/versions

markuskowa committed:
mono-zeroconf: add license

markuskowa committed:
msilbc: add license + description

danieldk committed:
eigen3_3: 3.3.4 -> 3.3.5

Changes: http://eigen.tuxfamily.org/index.php?title=ChangeLog#Eigen_3.3.5

markuskowa committed:
mythes: add license

Synthetica9 committed:
sway-beta: init at 1.0-beta1 (#48829)

* sway-beta: init at 1.0-beta1

* sway, sway-beta: use https url

primeos committed:
nixos/rootston: Remove the module and the package (#48905)

Rootston is just a reference compositor so it doesn't make that much
sense to have a module for it. Upstream doesn't really like it as well:

"Rootston will never be intended for downstream packages, it's an
internal thing we use for testing." - SirCmpwn [0]

Removing the package and the module shouldn't cause much problems
because it was marked as broken until
886131c2434dfce5e3c8a3584467ae914b482ce9. If required the package can
still be accessed via wlroots.bin (could be useful for testing
purposes).

[0]: https://github.com/NixOS/nixpkgs/issues/38344#issuecomment-378449256

primeos committed:
signal-desktop: 1.16.3 -> 1.17.0

nlewo committed:
Merge pull request #48578 from vdemeester/init-skaffold

skaffold: init at 0.16.0

markuskowa committed:
Merge pull request #48908 from danieldk/eigen-3.3.5

eigen3_3: 3.3.4 -> 3.3.5

7c6f434c committed:
Merge pull request #48890 from r-ryantm/auto-update/sleuthkit

sleuthkit: 4.6.2 -> 4.6.3

dywedir committed:
mako: init at 1.1 (#46606)

r-ryantm committed:
rdma-core: 20 -> 20.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/rdma-core/versions

xeji committed:
Merge pull request #48686 from dtzWill/update/libinput-gestures-2.38

libinput-gestures: 2.33 -> 2.38, fixup

Commit:
libgnurl: 7.54.1 -> 7.61.1. (#48792)

* libgnurl: 7.54.1 -> 7.61.1.

- fetch url updated (the mirror on gnunet.org seems no longer
maintained).
- most configure flags are not needed anymore since version 7.56.1 as
announced in the README
- flags -- without-ca-bundle and --with-ca-fallback are needed for
finding the CA certificates
- dependencies updated (copied from the guix package)
- license corrected

* libgnurl: fix idn support; use mirror url; add http2 support

timokau committed:
r8168: init at 8.046.00

royneary committed:
maintainers/maintainer-list: add royneary

royneary committed:
git-bug: init at 0.4.0

Add git-bug, a decentralized bug tracker written in Go

r-ryantm committed:
plantuml: 1.2018.11 -> 1.2018.12

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/plantuml/versions

markuskowa committed:
Merge pull request #48915 from r-ryantm/auto-update/rdma-core

rdma-core: 20 -> 20.1

r-ryantm committed:
pg_repack: 1.4.3 -> 1.4.4

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/pg_repack/versions

royneary committed:
git-bug: install man pages

NeQuissimus committed:
btrfs-progs: 4.15.1 -> 4.17.1

Fuuzetsu committed:
Merge pull request #48882 from r-ryantm/auto-update/snd

snd: 18.7 -> 18.8

jwiegley committed:
coqPackages.Velisarios: New expression

Commit:
QuadProgpp: Add new package.

Necessary for #48895.

Ericson2314 committed:
Merge pull request #47430 from roberth/nixpkgs-extend

pkgs.extend for adding overlays

Commit:
QuadProgpp: Remember to add to all-packages.nix

Commit:
Merge remote-tracking branch 'origin/master'

r-ryantm committed:
python36Packages.xdot: 0.9 -> 1.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-xdot/versions

Ericson2314 committed:
Merge remote-tracking branch 'upstream/master' into darwin-to-linux-cross

r-ryantm committed:
python36Packages.stem: 1.6.0 -> 1.7.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-stem/versions

r-ryantm committed:
python36Packages.txtorcon: 18.0.2 -> 18.3.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-txtorcon/versions

r-ryantm committed:
python36Packages.warlock: 1.2.0 -> 1.3.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-warlock/versions

r-ryantm committed:
python36Packages.python-sql: 0.9 -> 1.0.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-python-sql/versions

r-ryantm committed:
python36Packages.XlsxWriter: 1.1.1 -> 1.1.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-xlsxwriter/versions

r-ryantm committed:
python36Packages.simpleeval: 0.9.6 -> 0.9.8

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-simpleeval/versions

r-ryantm committed:
python36Packages.smart_open: 1.6.0 -> 1.7.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-smart_open/versions

r-ryantm committed:
python36Packages.twilio: 6.8.0 -> 6.19.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-twilio/versions

r-ryantm committed:
python36Packages.python-jsonrpc-server: 0.0.1 -> 0.0.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-python-jsonrpc-server/versions

r-ryantm committed:
python36Packages.python-utils: 2.2.0 -> 2.3.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-python-utils/versions

r-ryantm committed:
python36Packages.sortedcollections: 0.4.2 -> 1.0.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-sortedcollections/versions

danieldk committed:
fzf: 0.17.4 -> 0.17.5

Changes: https://github.com/junegunn/fzf/blob/master/CHANGELOG.md

r-ryantm committed:
python36Packages.validictory: 1.0.0a2 -> 1.1.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-validictory/versions

r-ryantm committed:
python36Packages.transaction: 2.2.1 -> 2.3.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-transaction/versions

r-ryantm committed:
python36Packages.qtpy: 1.4.2 -> 1.5.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-qtpy/versions

r-ryantm committed:
python36Packages.typing-extensions: 3.6.5 -> 3.6.6

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-typing_extensions/versions

r-ryantm committed:
python36Packages.shellingham: 1.2.6 -> 1.2.7

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-shellingham/versions

r-ryantm committed:
python36Packages.rebulk: 0.9.0 -> 1.0.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-rebulk/versions

r-ryantm committed:
python36Packages.rply: 0.7.5 -> 0.7.6

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-rply/versions

r-ryantm committed:
python36Packages.scp: 0.11.0 -> 0.12.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-scp/versions

r-ryantm committed:
python36Packages.statsd: 3.2.1 -> 3.3.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-statsd/versions

c0bw3b committed:
Merge pull request #48553 from r-ryantm/auto-update/aws-sdk-cpp

aws-sdk-cpp: 1.5.17 -> 1.6.20

r-ryantm committed:
python36Packages.schedule: 0.3.2 -> 0.5.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-schedule/versions

r-ryantm committed:
python36Packages.yamllint: 1.11.1 -> 1.12.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-yamllint/versions

r-ryantm committed:
python27Packages.gnutls: 3.0.0 -> 3.1.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python2.7-python-gnutls/versions

r-ryantm committed:
python36Packages.vcrpy: 1.13.0 -> 2.0.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-vcrpy/versions

r-ryantm committed:
python36Packages.pyRFC3339: 0.2 -> 1.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-pyrfc3339/versions

r-ryantm committed:
python36Packages.rpy2: 2.9.3 -> 2.9.4

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-rpy2/versions

r-ryantm committed:
python36Packages.qtawesome: 0.4.4 -> 0.5.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-qtawesome/versions

r-ryantm committed:
python36Packages.rx: 1.6.0 -> 1.6.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-rx/versions

r-ryantm committed:
python36Packages.readme_renderer: 21.0 -> 22.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-readme_renderer/versions

r-ryantm committed:
python36Packages.prawcore: 0.15.0 -> 1.0.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-prawcore/versions

r-ryantm committed:
python36Packages.musicbrainzngs: 0.5 -> 0.6

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-musicbrainzngs/versions

r-ryantm committed:
python36Packages.node-semver: 0.3.0 -> 0.4.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-node-semver/versions

r-ryantm committed:
python36Packages.kazoo: 2.2.1 -> 2.5.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-kazoo/versions

r-ryantm committed:
python36Packages.pylama: 7.5.5 -> 7.6.5

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-pylama/versions

Mic92 committed:
Merge pull request #48941 from r-ryantm/auto-update/python3.6-python-jsonrpc-server

python36Packages.python-jsonrpc-server: 0.0.1 -> 0.0.2

r-ryantm committed:
python36Packages.mahotas: 1.4.2 -> 1.4.5

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-mahotas/versions

LnL7 committed:
yarn: add platforms (#48912)

r-ryantm committed:
python36Packages.nipy: 0.4.0 -> 0.4.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-nipy/versions

xeji committed:
Merge pull request #48909 from markuskowa/licenses-23

Add licenses

veprbl committed:
arrow-cpp: fix for zstd 1.3.6+ (#48928)

gnidorah committed:
mytetra: support opening urls under wayland (#48641)

r-ryantm committed:
python36Packages.podcastparser: 0.6.2 -> 0.6.4

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-podcastparser/versions

Synthetica9 committed:
atom, atom-beta: move versions to top of file

Synthetica9 committed:
atom, atom-beta: 1.31.2 -> 1.32.0, 1.32.0-beta0 -> 1.33.0-beta0

r-ryantm committed:
python27Packages.mechanize: 0.3.5 -> 0.3.7

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python2.7-mechanize/versions

thanegill committed:
Airtame 3.1.1 -> 3.3.0 (#48505)

adisbladis committed:
firefox-devedition-bin: 63.0b14 -> 64.0b3

adisbladis committed:
firefox-beta-bin: 63.0b14 -> 64.0b3

timokau committed:
Merge pull request #48921 from timokau/r8168

r8168: init at 8.046.00

r-ryantm committed:
python36Packages.lmdb: 0.92 -> 0.94

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-lmdb/versions

xeji reviewed (changes requested):
I see a lot of commits that don't belong here. Please rebase your branch on current master to get rid of them.

Mic92 committed:
Merge pull request #48976 from r-ryantm/auto-update/python3.6-podcastparser

python36Packages.podcastparser: 0.6.2 -> 0.6.4

LnL7 committed:
Merge pull request #48944 from danieldk/fzf-0.17.5

fzf: 0.17.4 -> 0.17.5

das-g committed:
tealdeer: 1.0.0 -> 1.1.0 (#48918)

and adapt description to changed upstream README.

r-ryantm committed:
python36Packages.kerberos: 1.2.4 -> 1.3.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-kerberos/versions

xeji committed:
Merge pull request #48924 from royneary/git-bug

git-bug: init at 0.4.0

r-ryantm committed:
python36Packages.outcome: 0.1.0 -> 1.0.0 (#48978)

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-outcome/versions

r-ryantm committed:
python36Packages.pudb: 2016.2 -> 2018.1 (#48975)

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-pudb/versions

r-ryantm committed:
python36Packages.persistent: 4.4.2 -> 4.4.3

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-persistent/versions

xeji committed:
Merge pull request #48979 from Synthetica9/atom-1.32.0

atom, atom-beta: 1.31.2 -> 1.32.0, 1.32.0-beta0 -> 1.33.0-beta0

r-ryantm committed:
twa: 1.5.1 -> 1.6.0 (#48900)

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/twa/versions

r-ryantm committed:
python36Packages.meld3: 1.0.0 -> 1.0.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-meld3/versions

r-ryantm committed:
python36Packages.jsonpointer: 1.9 -> 2.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-jsonpointer/versions

r-ryantm committed:
python36Packages.limnoria: 2016.05.06 -> 2018.09.09

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-limnoria/versions

r-ryantm committed:
python36Packages.Nuitka: 0.5.25 -> 0.6.0.4

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-nuitka/versions

r-ryantm committed:
python36Packages.keepkey: 4.0.0 -> 4.0.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-keepkey/versions

r-ryantm committed:
python36Packages.poyo: 0.4.0 -> 0.4.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-poyo/versions

r-ryantm committed:
python36Packages.powerline: 2.6 -> 2.7

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-powerline/versions

r-ryantm committed:
python36Packages.pycountry: 1.17 -> 18.5.26

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-pycountry/versions

r-ryantm committed:
python36Packages.pydub: 0.22.1 -> 0.23.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-pydub/versions

r-ryantm committed:
python36Packages.MechanicalSoup: 0.10.0 -> 0.11.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-mechanicalsoup/versions

r-ryantm committed:
python36Packages.plotly: 3.1.1 -> 3.3.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-plotly/versions

r-ryantm committed:
python36Packages.filebytes: 0.9.12 -> 0.9.17

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-filebytes/versions

Mic92 committed:
Merge pull request #48813 from loskutov/master

opensc: 0.18.0 -> 0.19.0

r-ryantm committed:
python27Packages.fudge: 1.1.0 -> 1.1.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python2.7-fudge/versions

r-ryantm committed:
python36Packages.geojson: 2.4.0 -> 2.4.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-geojson/versions

r-ryantm committed:
python36Packages.ezdxf: 0.8.1 -> 0.8.8

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-ezdxf/versions

timokau committed:
ntl: 11.3.0 -> 11.3.1

"Fixed a bug that effected image, kernel, and gauss routines for
Mat<zz_p>. These routines did not behave correctly when the input matrix
was zero. Also improved the mat_lzz_pTest program."

r-ryantm committed:
python27Packages.enum: 0.4.4 -> 0.4.7

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python2.7-enum/versions

r-ryantm committed:
python36Packages.django_nose: 1.4.4 -> 1.4.5

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-django-nose/versions

r-ryantm committed:
python36Packages.ipyparallel: 6.2.2 -> 6.2.3

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-ipyparallel/versions

r-ryantm committed:
python36Packages.django_polymorphic: 2.0.2 -> 2.0.3

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-django-polymorphic/versions

r-ryantm committed:
python36Packages.d2to1: 0.2.12 -> 0.2.12.post1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-d2to1/versions

r-ryantm committed:
python36Packages.colored: 1.1.5 -> 1.3.93

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-colored/versions

Ericson2314 committed:
Merge pull request #46534 from obsidiansystems/darwin-to-linux-cross

misc pkgs: Darwin -> Linux cross compilation

timokau committed:
Merge pull request #49008 from timokau/ntl-11.3.1

ntl: 11.3.0 -> 11.3.1

joachifm committed:
Merge pull request #48812 from mpickering/cedille

cedille bug fixes

mmcalistair committed:
Merge branch 'homely' of github.com:lokke-me/nixpkgs into homely
